views/apartments/delete.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>TUNR</title>
+  </head>
+  <body>
+    <h1>Delete <%=@apartment.name%>?</h1>
+<form action="/apartments/<%= @apartment.id %>" method="post">
+  <input type="hidden" name="_method" value="delete">
+  <input type="submit" value="Delete!">
+</form>
+
+  </body>
+</html>
```

Comment:
while this is fine, it doesn't have to be this way. Really you can put this form wherever. Whats important it the action method and hidden input for method in the form.